tests/vmi.json

```diff
@@ -0,0 +1,140 @@
+{
```

Comment:
@nunnatsa Hi. Is this a leftover or a required file?

Reply:
oops...